test/demo/utils/jestSetupAfterEnv.ts

```diff
@@ -0,0 +1,2 @@
+// Apps are taking a very long time to spawn on CI causing timeouts
```

Comment:
Suggest change file name to `integrationSetupAfterEnv`, for consistency with fdc3, and `integrationTestEnv.js`

Reply:
I've also gone lower with the timeout in FDC3 but defined a constant in` demo/constants.ts`, so it's easy to do something like, set a test that starts 2 apps to have a timeout of `appStartupTime * 2`. Worth copying?

Reply:
Wow, I really messed up that name. I was intending for it to be the same as FDC3.

I copied the content from fdc3, but must have an old version locally. Happy to change as long as it works.

Reply:
As for the `constants.ts` suggestion, I think I'd rather just leave this all self contained for now. Our tests are generally very lightweight and I'd rather remove this entirely once the test runner is fixed.